src/verifiable/credential.ts

```diff
@@ -5,14 +5,20 @@ import {
   CredentialStatus,
   RefreshService
 } from './proof';
-import { Claim, DID } from '@iden3/js-iden3-core';
-import { ProofType } from './constants';
+import {
+  Claim,
+  DID,
+  MerklizedRootPosition as MerklizedRootPositionCore,
+  IdPosition
+} from '@iden3/js-iden3-core';
 import { Proof, Hash, rootFromProof, verifyProof } from '@iden3/js-merkletree';
 import { Merklizer, Options } from '@iden3/js-jsonld-merklization';
 import { PublicKey, poseidon } from '@iden3/js-crypto';
 import { CredentialStatusResolverRegistry } from '../credentials';
 import { getUserDIDFromCredential } from '../credentials/utils';
 import { validateDIDDocumentAuth } from '../utils';
+import { Parser, CoreClaimOptions } from '../schema-processor/json';
```

Comment:
seems like a circular dependecy? @Kolezhniuk 

can't we do it as in go ?

Reply:
@vmidyllic ` No circular dependency found!`